backend/src/main/java/com/redhat/cloud/notifications/auth/principal/rhid/RhIdentity.java

```diff
@@ -1,44 +1,12 @@
 package com.redhat.cloud.notifications.auth.principal.rhid;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.redhat.cloud.notifications.auth.principal.ConsoleIdentity;
 
-@JsonIgnoreProperties(ignoreUnknown = true)
-public class RhIdentity extends ConsoleIdentity {
+public abstract class RhIdentity extends ConsoleIdentity {
 
-    @JsonProperty("account_number")
-    private String accountNumber;
-
-    @JsonProperty("org_id")
-    private String orgId;
-
-    private User user;
+    public abstract String getOrgId();
 
     public String getAccountNumber() {
-        return accountNumber;
-    }
-
-    public User getUser() {
-        return user;
-    }
-
-    public String getOrgId() {
-        return orgId;
-    }
-
-    @JsonIgnoreProperties(ignoreUnknown = true)
-    public static class User {
-        @JsonProperty("username")
-        private String username;
-
-        public String getUsername() {
-            return username;
-        }
-    }
-
-    @Override
-    public String getName() {
-        return getUser().getUsername();
+        return null;
```

Comment:
Is that method really needed in this abstract class?

Reply:
Yes because account number it's still use on some table on db (we insert it but we never read it).
I think it should be remove on another ticket, with all remaining code about account number (RHCLOUD-30644).